README.md

```diff
@@ -5,7 +5,8 @@
 ### Building:
 
 ```bash
-git clone https://github.com/opencontainers/runc
+git clone https://github.com/opencontainers/runc $GOPATH/src/github.com/opencontainers/runc
```

Comment:
@stevvooe 
> This doesn't work for all GOPATH setups.

Could you please specify why it will not work for all `GOPATH` setups?


> I'd recommend go get -d to do a download only fetch into the GOPATH.

As I mentioned, I noticed that the Godeps directory is existed in the repository, and the go get is slow and unreliable, as it does not provide repeatable builds.